content/en/docs/instrumentation/js/automatic/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 30
+---
+
+Automatic instrumentation provides a way to instrument any Node.js application
+and capture telemetry data from many popular libraries and frameworks without
+any code changes.
+
+## Setup
+
+Run the following commands to install the appropriate packages.
+
+```shell
+npm install --save @opentelemetry/api
```

Comment:
One more nit - I don't think it's required to install the API separately, the instrumentation packages will pull it in.

Reply:
Actually I might be wrong on that since it is a peer dependency, but something is pulling it in. Probably fine to leave this there.

Reply:
Something is pulling it in, but that shouldn't be happening as far as I know.

Reply:
You are right ... it's the `api-logs` package through `otlp-transformer`. This needs to be fixed. But your example is correct, sorry about the false alarm.